src/pages/en/reference/api-reference.md

```diff
@@ -307,6 +307,23 @@ And would render HTML like this:
 </ul>
 ```
 
+### `Astro.generator`
+
+`Astro.generator` is a convenient way to add a [`<meta name="generator">`](https://html.spec.whatwg.org/multipage/semantics.html#meta-generator) tag with your current version of Astro. `Astro.generator` follows the format `Astro v0.0.0` where `v0.0.0` matches your current version of Astro.
+
+```astro
```

Comment:
🦛 would want you to know we can highlight the important bits here

````suggestion
```astro mark="Astro.generator"
````

But otherwise LGTM — thanks Nate, hadn’t even noticed that PR get merged 🙌

Reply:
D'oh! Even **I** am not used to pointing out our new code syntax yet... that's today's main task!